theme/static/css/hydroshare_core.css

```diff
@@ -1369,6 +1497,25 @@ body > .main-container {
 	color: #D9534F;
 }
 
+.custom-btn-toolbar.toolbar-fixed {
+	position: fixed;
+	background-color: #FFF;
+	border: 1px solid #DDD;
+	padding: 10px;
+	border-radius: 4px;
+	z-index: 3;
+}
+
+.custom-btn-toolbar {
+    display: flex;
```

Comment:
spaces here and tabs above?  I'm seeing this all over this css file... it's probably elsewhere too.  I think we need to run these files through a lint.

Reply:
I agree. We should probably make a separate ticket for this and enforce lint after that.

Reply:
That's fine, please create the ticket so we can get it prioritized.